en/conduct/index.md

```diff
@@ -16,7 +16,6 @@ community. It applies to all "collaborative space", which is defined as
 community communications channels (such as mailing lists, submitted patches,
 commit comments, etc.).
 
- * Participants will be tolerant of opposing views.
  * Participants must ensure that their language and actions are free of personal attacks and disparaging personal remarks.
  * Participants should speak and act with good intentions, but understand that intent and impact are not equivalent.
- * Behaviour which can be reasonably considered harassment will not be tolerated.
+ * Behaviour which can be considered harassment against protected classes will not be tolerated.
```

Comment:
What are the protected classes? Who decides? is harassment allow against non-protected classes?

Reply:
@Try2Code see https://github.com/ruby/www.ruby-lang.org/pull/2690#issuecomment-931278138

Reply:
yh thx, I already put my concerns in the PR ;-)
a lot better for reasonable conversation :+1: